sumy/nlp/tokenizers.py

```diff
@@ -77,6 +77,23 @@ def tokenize(self, text):
         return kkma.nouns(text)
 
 
+class GreekSentencesTokenizer:
+    """Calls sent_tokenize for greek, which doesn't split sentences on semicolon ';' 
+    or the greek question mark ';'. The regex below splits on both.
+    """
+    @classmethod
+    def tokenize(self, text):
+        sentences = nltk.sent_tokenize(text, language = 'greek')
+        sentences = [list(filter(None, re.split(r'(?<=[;;])\s+', sentence))) for sentence in sentences]
```

Comment:
Can you explain the regex? It seems it's more complicated than it has to be. Is the regex below equivalent?

```suggestion
        sentences = [list(filter(None, re.split(r'(?<=;)\s+', sentence))) for sentence in sentences]
```